data/remnant missions.txt

```diff
@@ -1713,3 +1717,150 @@ mission "Remnant: Broken Jump Drive job"
+			`	"As the primary example of space-faring life, the void sprites have always been of interest to me. They have numerous advantages that we as atmosphere-bound creatures can only dream of, such as being able to experience space in a much more direct manner. I have been working on a side project to build a new generation of ships that melds these ideas with our current technology, but the data we have on the void sprites is centuries old and incomplete," her chant trails off as she pulls up a display with your original scans on them. "However, your recent explorations provided new data I could use to update my designs. This slime that you retrieved is the final ingredient."`
+			`	The machine dings, and she looks over the results and then trills excitedly. "Yes, it matches! So it is time to begin the final activation." She strides over to a central console and taps in a series of commands, followed by an iris scan and a blood prick. The industrial hum of the room suddenly ramps up to a crescendo as your nose is assaulted by what you can only describe as the smell of a thunderstorm in a swamp. A few seconds later the noise softens, and the tank starts lowering into the floor as a shape emerges from the liquid.`
+			`	The ship before you has some faint traces of the gascraft you have flown to Nenia, along with other elements of Remnant design, but it is clearly something new. Its antennae swivel around, as if taking in its surroundings, and the tentacles along the side twitch and reach out to touch its surroundings. Solidly built tendrils underneath it flex as it lifts itself off its supports and begins to move around. You have a momentary vision of what this ship might look like in the skies of Nenia.`
+			scene "scene/penguinscene"
+			`	Taely's droning chant of technical details finally seeps through your reverie: "... so from a captain's point of view flying the penguin should be very similar to a normal ship, except she flies herself and doesn't need a crew to run her systems." She pauses, and looks at you. "Your curiosity provided the final ingredients to bring her to life. You should be the first to have one. You'll still have to go through the shipyard to register her and pay the resource cost to balance the logistics, but this first one is reserved for you." She pats the nose of the penguin and it makes a wooshing noise. After a moment she starts going over the ship with a scanner, leaving you to contemplate in silence.`
```

Comment:
Even though Taely's droning is continuing on from somewhere else, "so" should be capitalized as part of normal dialogue rules, since an ellipsis is considered to be a punctuation mark.

We can't call the ship "the penguin" right away, since nobody's named it in dialogue yet.

Reply:
Capitalized the S.

In regards to calling it the penguin before anyone has named it yet, I did it that way to further the implication that the player (and the pilot) just missed a whole chunk of explanation that presumably included what Taely called it. That way it isn't just that we're *told* that we missed some of what she was saying, we have a concrete instance where the perceptive player can note that "hey, I missed where this got named," thus making the fact that they missed it more personal and not just something they are told.

Reply:
I didn't notice that Taely talked about the penguin in her dialogue, only when the narration said it.